package.json

```diff
@@ -44,6 +44,8 @@
   "devDependencies": {
     "babel-core": "6.6.0",
     "babel-eslint": "5.0.0",
+    "babel-istanbul": "^0.6.0",
```

Comment:
Also, with exact requirements it lets us use [greenkeeper](http://greenkeeper.io/) to test with new releases as they happen.